shards/egui/src/containers/docking.rs

```diff
@@ -51,7 +54,13 @@ lazy_static! {
     cstr!("The UI contents containing tabs."),
     &SHARDS_OR_NONE_TYPES[..],
   )
-    .into(),];
+    .into(),
+  (
+    cstr!("DefaultTab"),
+    cstr!("The index of the tab to open by default. 0 being the right most tab."),
+    &INT_OR_NONE_TYPES_SLICE[..],
```

Comment:
You can just put INT_TYPES (without none), since you're setting the default to 0